careers/careers/templates/careers/position.jinja

```diff
@@ -1,5 +1,6 @@
 {% extends "base.jinja" %}
 
+{% block meta_description %}{{ meta_description|safe }}{% endblock %}
```

Comment:
This shouldn't be marked `safe`, because it includes data directly from Greenhouse. Only `description` [gets bleached](https://github.com/mozmeao/lumbergh/blob/master/careers/careers/management/commands/sync_greenhouse.py#L75) when collection the data.

Alternatively I suggest that you use Jinja's [`truncate`](https://jinja.palletsprojects.com/en/2.10.x/templates/#truncate) filter.

```
{% block meta_description %}{{ meta_description|truncate(160, end='…') }}{% endblock %}
```

BTW I have a question: Why do we prefer to use `&hellip` instead of the actual utf-8 character?

Reply:
Got it, thanks!